### PR TITLE
Update CODE_OF_CONDUCT file

### PR DESCRIPTION
**Pull request by michaelld**
subject says it all: GR just updated their CoC ; this change brings the Volk CoC in closer alignment with the GR CoC. Please note that I tweaked the language with respect to "regardless of" because I hate that language as it always forgets some group or class of people. I hope I got the language better & also the MD markup correct! This PR is testing for both of those. I am not committed to the language, but I do hope we can find a better way to be fully inclusive rather than via some special list.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,80 +1,127 @@
...
+We as members, contributors, and leaders strive to make participation
+in our community a harassment-free experience for
+everyone. Period. Qualities and characteristics we note specifically
+include regardless of age, body size, visible or invisible disability,
+ethnicity, sex characteristics, gender identity, gender expression,
+sexual identity, sexual orientation, level of experience, education,
+socio-economic status, nationality, personal appearance, race, caste,
+color, religion, and military status -- but lists like these cannot be
+complete and hence we note "for everyone. Period." in order to be as
+inclusive as possible.
```
Comment (jdemel) [inline CODE_OF_CONDUCT.md]: I assume this is the paragraph where you added some things?
I'd say the goal is clear. It sounds right. Only, I'd remove the "Period." parts. These make it sound grumpy.

**Review by jdemel — COMMENTED** [review]
Thanks for taking care of this update. GR adapted its own CoC based on another source. I don't have the source handy at the moment.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (michaelld) [reply CODE_OF_CONDUCT.md]: OK & That's fine with me. Here is [the original GR source](https://github.com/gnuradio/gr-governance/blob/main/CODE_OF_CONDUCT.md).

**Review by jdemel — APPROVED** [review]
LGTM.
If there is anything missing or incorrect, we can always improve it.

**Pull request merged**